Tools/Dev/Design/README.md

```diff
@@ -30,22 +30,21 @@ The B.C. Government colour palette ensures all public facing government services
 #### Brand Colours
 
 <p float="left">
-    <img src="images/primary.png" alt="Primary Color" width="100" />
-    <img src="images/secondary.png" alt="Secondary Color" width="100" />
-    <img src="images/heading-text.png" alt="Heading Color" width="100" />
-    <img src="images/input-text.png" alt="Input Text Color" width="100" />
-    <img src="images/accent.png" alt="Accent Color" width="100" />
+    <img src="images/colours/primary.png" alt="Primary Color" width="100" />
```

Comment:
I dont think it would be a good idea to put the images under a colours folder since the images are not colours

Reply:
Are you saying you'd prefer something like `images/primary-colour.png`?  I'm not sure there's much difference, but I'm happy with either option.

Reply:
sorry about the confusion. My intention by the name of images was the actual image files that contained in design system. The images folder was there initially so I assumed that's how you prefer me doing them.

Reply:
@MikeLyttle Since not all images that are on the colors folder are actual colors but instead variants or styles.  Moreover, ```comment-icons.png``` does not sound like color at all to me.